apps/jetpack-cloud/webpack.config.js

```diff
@@ -0,0 +1,149 @@
+/**
+ **** WARNING: No ES6 modules here. Not transpiled! ****
+ */
+/* eslint-disable import/no-nodejs-modules */
+
+/**
+ * External dependencies
+ */
+const _ = require( 'lodash' );
+const FileConfig = require( '@automattic/calypso-build/webpack/file-loader' );
+const getBaseWebpackConfig = require( '@automattic/calypso-build/webpack.config.js' );
+const path = require( 'path' );
+const SassConfig = require( '@automattic/calypso-build/webpack/sass' );
+const TranspileConfig = require( '@automattic/calypso-build/webpack/transpile' );
+
+/**
+ * Internal variables
+ */
+const cacheIdentifier = require( '../../server/bundler/babel/babel-loader-cache-identifier' );
+const isDevelopment = process.env.NODE_ENV !== 'production';
+
+const nodeModulesToTranspile = [
+	// general form is <package-name>/.
+	// The trailing slash makes sure we're not matching these as prefixes
+	// In some cases we do want prefix style matching (lodash. for lodash.assign)
+	'@github/webauthn-json/',
+	'acorn-jsx/',
+	'chalk/',
+	'd3-array/',
+	'd3-scale/',
+	'debug/',
+	'escape-string-regexp/',
+	'filesize/',
+	'prismjs/',
+	'react-spring/',
+	'regenerate-unicode-properties/',
+	'regexpu-core/',
+	'striptags/',
+	'unicode-match-property-ecmascript/',
+	'unicode-match-property-value-ecmascript/',
+];
+
+/**
+ * Check to see if we should transpile certain files in node_modules
+ *
+ * @param {string} filepath the path of the file to check
+ * @returns {boolean} True if we should transpile it, false if not
+ *
+ * We had a thought to try to find the package.json and use the engines property
+ * to determine what we should transpile, but not all libraries set engines properly
+ * (see d3-array@2.0.0). Instead, we transpile libraries we know to have dropped Node 4 support
+ * are likely to remain so going forward.
+ */
+function shouldTranspileDependency( filepath ) {
```

Comment:
This is borrowed from Calypso's webpack config file. Ideally, this should be defined in a single place and reused in both locations.